README.md

```diff
@@ -4,364 +4,414 @@ The purpose of the ODM is to support wastewater-based surveillance and epidemiol
 
 The ODM supports data sharing in two ways:
 
-1. **Data sharing schema** - The ODM will have a schema that describes what data can be shared with one or more partners or users.
-2. **Data filter based on the data sharing schema** - The ODM will support an open source method for filtering data tables using the data sharing schema.
+1.  **Data sharing schema** - The ODM will have a schema that describes what data can be shared with one or more partners or users.
+2.  **Data filter based on the data sharing schema** - The ODM will support an open source method for filtering data tables using the data sharing schema.
 
-The data sharing schema will be a csv file (sharing.csv) where each row in the file corresponds to one variable in a ODM table. Attributes in the row describe who the data is shared with and what data is excluded. See below for an example.
+The data sharing schema will be a csv file (sharing.csv) where each row in the `sharing` file corresponds to a header or table in the PHES-ODM. Attributes in the row describe who the data is shared with and what data is included. See below for an example.
 
```

Comment:
Each row should have version and update fields, similar to all ODM tables: status,firstReleased,lastUpdated,changes

Reply:
Added in the most recent push.